executor/src/native_executor.rs

```diff
@@ -197,19 +198,17 @@ mod tests {
 		construct_block(
 			2,
 			block1().1,
-			hex!("2e69e4405a13981224078ad5355c68401bf56d0fe3f14a3536734666e6a8a047"),
+			H256(hex!("2cdbbf9bd766c2286a5f4091c131fe161addd060ba6fc041b3419089f4601bda")),
```

Comment:
maybe introduce an `h256!(s)` macro that evaluated to `H256(hex!(s))` to avoid the noise.